onebot/plugins/botui.py

```diff
@@ -132,6 +132,15 @@ def quote(self, mask, target, args):
         self.bot.send(cmd)
 
     @command(permission='admin', show_in_help_list=False)
+    def load(self, mask, target, args):
```

Comment:
Does this also work for plugins that are not already loaded? If not, there probably should be some error handling. If so, the help-text possibly needs to be updated.